scheduler/boot-scheduler-quartz/src/main/java/com/scheduler/quartz/web/controller/JobsController.java

```diff
@@ -35,7 +39,7 @@ List<JobStatus> getJobsStatuses() throws SchedulerException {
     }
 
     @PostMapping(value = "/saveOrUpdate")
-    public Message saveOrUpdate(ScheduleJob job) {
+    public Message saveOrUpdate(@RequestBody @Valid ScheduleJob job) {
```

Comment:
@coderabbitai, please help me to write integrationTests by extending `AbstractIntegrationTest` for all endpoints in this controller